rx3-java/rx3grpc-stub/src/test/java/com/salesforce/rx3grpc/stub/RetryWhen.java

```diff
@@ -198,9 +197,15 @@ public static Builder exponentialBackoff(long firstDelay, TimeUnit unit) {
 
     public static final class Builder {
 
+        static final class TruePredicate implements Predicate<Object> {
+            @Override
+            public boolean test(Object o) {
+                return true;
+            }
+        }
         private final List<Class<? extends Throwable>> retryExceptions = new ArrayList<Class<? extends Throwable>>();
         private final List<Class<? extends Throwable>> failExceptions = new ArrayList<Class<? extends Throwable>>();
-        private Predicate<? super Throwable> exceptionPredicate = Functions.alwaysTrue();
+        private Predicate<? super Throwable> exceptionPredicate = new TruePredicate();
```

Comment:
Could this just be `private Predicate<? super Throwable> exceptionPredicate = x -> true;`